app/robot-shop/helm/values.yaml

```diff
@@ -97,7 +97,15 @@ dispatch:
   nodeSelector:
     workload: app
 
-mongodb: {}
+mongodb:
```

Comment:
- change node-group `prod-cluster` -> `app-ng`
- change namespace `prod-robot-shop` -> `robot-shop`
  - `change references to cluster name across the repo, `{makefile, helm values, bash scripts}`
- change cluster name: `prod-eks-cluster` -> `sre-stack`
  - `change references to cluster name across the repo, `{makefile, helm values, bash scripts}`